tendrl/monitoring_integration/graphite/__init__.py

```diff
@@ -146,7 +146,9 @@ def get_central_store_data(self, objects):
                         integration_id) + "/is_managed"
                     cluster_is_managed = etcd_utils.read(
                         cluster_key).value
-                    if cluster_is_managed.lower() == "no":
+                    if not cluster_is_managed or \
```

Comment:
I feel you can have the check like 
`if cluster_is_managed in [None, "", "no", "NO"]`

Reply:
i will change this